SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -157,6 +158,7 @@
   @JsonProperty
   private String dockerPrefix = "se-";
 
+  @NotNull
```

Comment:
`int` can't be null, but could could default to zero -- we should decide what a good minimum is, and then use the `@Min` annotation